tests/utils.py

```diff
@@ -132,6 +132,25 @@ def ensure_test_table(table_spec, target_db='postgres'):
             LOGGER.info("create table sql: %s", sql)
             cur.execute(sql)
 
+def build_alter_table_sql(table, col_spec):
+    sqls = []
+    if altered_name:=col_spec.get('change_name'):
+        sqls.append("ALTER TABLE {} RENAME COLUMN {} TO {}".format(table, col_spec['name'], altered_name))
+    if altered_type:=col_spec.get('is_change_type'):
+        sqls.append("ALTER TABLE {} ALTER COLUMN {} TYPE {}".format(table, col_spec['name'], altered_type))
+    if col_spec.get("is_new_col"):
+        sqls.append("ALTER TABLE {} ADD {} {}".format(table, col_spec['name'], col_spec['type']))
+    return sqls
```

Comment:
**Nits:** I understand that we will test more cases, but we don't actually use them all now nor the future. Thus, I believe that we should keep what we only need for now and add more utility in latter PR 👍

Reply:
The reason why I did this is because of the reason why you've specified. So If we only care about the adding another column may be we can remove this function and merge the logic in `add_columns ` function 😄